.circleci/config.yml

```
@@ -62,6 +62,26 @@ jobs:
           command: |
             coverage run  --source=. -m unittest discover tests
             bash <(curl -s https://codecov.io/bash) -cF python
+  lint-js:
+    <<: *defaults
+    steps:
+      - checkout
+      -run:
+        name: Yarn install
+        command: yarn install
+      - run: Lint js
+        command: yarn lint-js
+  test-js:
+    <<: *defaults
+    steps:
+      - checkout
+      -run:
```

This needs a space

```suggestion
      - run:
```

And in the one above too.

---

See https://app.circleci.com/pipelines/github/canonical-web-and-design/cn.ubuntu.com/556/workflows/c1079fd6-7a43-475c-b63a-9f052ff451ab/jobs/2022/steps